modules/issue_tracker/php/NDB_Menu_Filter_Form_issue_tracker.class.inc

```diff
@@ -41,19 +41,38 @@ class NDB_Menu_Filter_Form_Issue_Tracker extends NDB_Menu_Filter_Form
         $user =& User::singleton();
         $db   = Database::singleton();
 
+        //Why is this not a multi-line string?
```

Comment:
You can leave these questions in the PR description and bring them up for discussion instead of polluting the code